src/database/models/version_item.rs

```diff
@@ -84,6 +84,17 @@ impl VersionBuilder {
 
         version.insert(&mut *transaction).await?;
 
+        sqlx::query!(
+                "
+                UPDATE mods
+                SET updated = NOW()
+                WHERE id = $1
+                ",
+                self.mod_id as ModId,
+            )
+            .execute(&mut *transaction)
+            .await?;
```

Comment:
The indentation is messed up here; fix it and run `cargo fmt` to get the rest of the formatting issues.